includes/multi-currency/CountryFlags.php

```diff
@@ -288,12 +289,12 @@ public static function get_by_country( string $country ): string {
 	 */
 	public static function get_by_currency( string $currency ): string {
 		$exceptions = [
-			'ANG' => '',
-			'BTC' => '',
-			'XAF' => '',
-			'XCD' => '',
-			'XOF' => '',
-			'XPF' => '',
+			Currency_Code::NETHERLANDS_ANTILLEAN_GUILDER => '',
+			Currency_Code::BITCOIN                       => '',
+			Currency_Code::CENTRAL_AFRICAN_CFA_FRANC     => '',
+			Currency_Code::EAST_CARIBBEAN_DOLLAR         => '',
+			Currency_Code::WEST_AFRICAN_CFA_FRANC        => '',
+			Currency_Code::CFP_FRANC                     => '',
```

Comment:
@jessy-p @zmaglica please double check the value of this constant in the new class. I just found a bug because of value was different.

Reply:
I checked all these constants and it appears that the value is correct.